src/meeting/teams/speakersObserver.ts

```
@@ -0,0 +1,522 @@
+            } catch (error) {
+                console.error('[Teams] ❌ Error in speakers callback:', error)
+            }
+        })
+
+        // Inject EXACT SAME LOGIC as extension but via Playwright
+        await this.page.evaluate(
+            ({ recordingMode, botName, speakerLatency, mutationDebounce, checkInterval, freezeTimeout }) => {
+                console.log('[Teams-Browser] Setting up observation - EXACT EXTENSION LOGIC')
+
+                // EXACT SAME VARIABLES AS EXTENSION
+                let CUR_SPEAKERS = new Map<string, boolean>()
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'CUR_SPEAKERS' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)